src/WPF/ArcGISRuntime.WPF.Viewer/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml

```diff
@@ -14,7 +14,7 @@
                     Source="http://services.arcgisonline.com/ArcGIS/rest/services/World_Topo_Map/MapServer"/>
             </esri:Map>
         </esri:MapView>
-        <Border Name="uiPanel" 
+        <Border Name="UiPanel" 
```

Comment:
The "Name" attribute does nothing (it's not even "x:Name").

Reply:
trivia: I was surprised to learn that `Name` actually works. From SO: https://stackoverflow.com/questions/589874/#593151

We should be consistent, so I'll fix this everywhere. In this case the control isn't referenced at all, so I'll get rid of it.